File: backend-python/database_connection/create_database.py
import pymongo
from pymongo import MongoClient
def create_database():
    myclient = MongoClient("mongodb+srv://navjotdadwal:2019UEC1657@cluster0.lbdurre.mongodb.net/?retryWrites=true&w=majority")
    mydb = myclient["Flipr"]
    mycol = mydb["Admin Collection"]
    record = { "_id": 5,
          "name": "Raju",
          "Roll No": "1005",
          "Branch": "CSE"}
    rec_id1 = mycol.insert_one(record)

create_database()

File: node-backend/database_connection/create_database.js
async function main() {
    const mongoString = "mongodb+srv://navjotdadwal:2019UEC1657@cluster0.lbdurre.mongodb.net/?retryWrites=true&w=majority"

    mongoose.connect(mongoString, {useNewUrlParser: true})

    mongoose.connection.on("error", function(error) {
    console.log(error)
    })

    mongoose.connection.on("open", function() {
    console.log("Connected to MongoDB database.")
    })
}
async function listDatabases(client){
    databasesList = await client.db().admin().listDatabases();
 
    console.log("Databases:");
    databasesList.databases.forEach(db => console.log(` - ${db.name}`));
};
main().catch(console.error);
